src/samples/audio/smart_amp_test_ipc4.c

```diff
@@ -65,7 +74,9 @@ static int smart_amp_init(struct processing_module *mod)
 
 	if (base_cfg->base_cfg_ext.nb_input_pins != SMART_AMP_NUM_IN_PINS ||
 	    base_cfg->base_cfg_ext.nb_output_pins != SMART_AMP_NUM_OUT_PINS) {
-		comp_err(dev, "smart_amp_init(): Invalid pin configuration");
+		comp_err(dev, "smart_amp_init(): Invalid pin configuration: %d %d",
```

Comment:
I would keep changes to bare minimum needed in any module. Chnages to log messages should be another PR.